Directory.Packages.props

```diff
@@ -195,7 +195,7 @@
     <PackageVersion Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
 
     <!-- Pinned versions for Component Governance - Remove when root dependencies are updated -->
-    <PackageVersion Include="System.Text.Json" Version="8.0.4" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.5" />
```

Comment:
This was pinned for CG so I'm hesitant to keep it as 8.0.4. The intention would be that at some point, whoever is bringing that transitively would update themselves to something past 8.0.4 and we can remove the transitive pinning. If we keep bumping this, then that is unlikely to happen.

Reply:
8.0.4 is now marked vulnerable

https://www.nuget.org/packages/System.Text.Json/8.0.4

![image](https://github.com/user-attachments/assets/ab791b39-d9ef-4cfc-a6a6-20ca8a67d2dd)

Reply:
gotcha, ok that totally makes sense then, sorry wasn't aware.